packages/terra-search-field/src/SearchField.jsx

```diff
@@ -306,13 +305,12 @@ class SearchField extends React.Component {
         && (
         <label className={cx('label')}>{groupName}</label>
         )}
-        <div role="search" className={cx('search-role-container')} aria-label={intl.formatMessage({ id: 'Terra.searchField.search' })}>
-          <div role="group" aria-label={groupNameValue} {...customProps} className={searchFieldClassNames}>
+        <div className={cx('search-role-container')}>
+          <div {...customProps} className={searchFieldClassNames}>
             <div className={cx('input-group')}>
               <input
                 {...additionalInputAttributes}
                 className={inputClass}
-                type="search"
```

Comment:
I feel even this should not be removed . See Here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search. It should announce like this. Changing default aria label (groupName default value) should be the way to go.

Reply:
Added.